android/README.md

```diff
@@ -85,7 +89,9 @@ library.
     linker = "${your NDK x.y.z path}/toolchains/llvm/prebuilt/darwin-x86_64/bin/armv7a-linux-androideabi35-clang"
     ```
 
-6. Run `rustup target add armv7-linux-androideabi aarch64-linux-android`
-7. Open `convex-mobile/android` in Android Studio (wait for it to sync)
-8. Double press Ctrl and type `./gradlew build`
-9. You can generate a local Maven installable package by running `./gradlew publishToMavenLocal`
\ No newline at end of file
+7. Run `rustup target add armv7-linux-androideabi aarch64-linux-android`
+8. Open `convex-mobile/android` in Android Studio (wait for it to sync)
+9. Double press Ctrl and type `./gradlew build`
+10. Set up GPG key/passphrase https://central.sonatype.org/publish/requirements/gpg/
```

Comment:
Too bad this is required just to build. It would be nice to not require GPG to have someone just work on the library. Since that's not how it is though, it's nice that this is at least documented now though!